.github/workflows/continuous-deployment.yml

```diff
@@ -34,6 +34,9 @@ jobs:
       - name: Change wrapper permissions
         run: chmod +x ./gradlew
 
+      - name: Generate docs
+        run: ./gradlew dokkaHtmlMultiModule
+
```

Comment:
I think this generates documentation but uploads nowhere, maybe we can remove it and integrate in the publication plugin in another issue?

Reply:
updated